pkg/universe.dagger.io/python/test/test.cue

```diff
@@ -3,13 +3,28 @@ package python
 import (
 	"dagger.io/dagger"
 	"dagger.io/dagger/core"
-
+	"universe.dagger.io/docker"
 	"universe.dagger.io/python"
 )
 
 dagger.#Plan & {
 	actions: test: {
 
+		// Run with a custom path to python
+		customPath: {
+			
+			// python:3.10-alpine has both python3 and python on the PATH
+			_image: docker.#Pull & {
```

Comment:
That wouldn't be better to have a `python.#Image`? Like we do with `go.#Image`?

Reply:
That's a good idea, but I may defer it because I have another change I want to make to the default image (see my comment to @helderco above and the small discussion in #universe on discord,